modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -440,7 +440,7 @@
     ::upload/int                      [:bigint]
     ;; identity(1, 1) defines an auto-increment column starting from 1
     ::upload/auto-incrementing-int-pk [:bigint [:identity 1 1]]
-    ::upload/float                    [:float]
+    ::upload/float                    [(keyword "double precision")]
```

Comment:
float is an alias for double precision, but double precision is what you get when you do `show table` so it seems more appropriate